clang/include/clang/CIR/Dialect/Passes.td

```diff
@@ -93,4 +93,23 @@ def LoweringPrepare : Pass<"cir-lowering-prepare"> {
   let dependentDialects = ["cir::CIRDialect"];
 }
 
+def LibOpt : Pass<"cir-lib-opt"> {
+  let summary = "Optimize C/C++ library calls";
+  let description = [{
+    By using higher level information from `cir-idiom-recognize`, this pass
+    apply transformations to CIR based on specific C/C++ library semantics.
+
+    Transformations done by this pass can be inspected by users by using
+    remarks. Currently supported are `all` and `transforms`.
```

Comment:
I don't think this is even true yet in the incubator implementation. I would omit this paragraph until it's true.